### ISSUE TITLE
index.android.js: Unexpected character '​'

### ISSUE THREAD

**Issue by hashedi** [GeekyAnts/NativeBase#530]
import React, { Component } from 'react';
import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Body, Icon } from 'native-base';
​export default class AnatomyExample extends Component {
    render() {
        return (
          <Container>
              <Header>
                  <Left>
                      <Button transparent>
                          <Icon name='menu' />
                      </Button>
                  </Left>
                  <Body>
                      <Title>Header</Title>
                  </Body>
                  <Right />
              </Header>
              <Content>
                  // Your main content goes here
              </Content>
              <Footer>
                  <FooterTab>
                      <Button full>
                          <Text>Footer</Text>
                      </Button>
                  </FooterTab>
              </Footer>
          </Container>
        );
    }
}

AppRegistry.registerComponent('TestBase', () => AnatomyExample);

"native-base": "^2.0.7",
    "react": "15.4.2",
    "react-native": "0.41.2"

I just start work with reactbase

**Comment by sankhadeeproy007** [comment]
Please type in the example instead of copy/pasting. Sometimes some rogue characters get carried over.

**Comment by MintyOwl** [comment]
![nb-error](https://cloud.githubusercontent.com/assets/7389244/23392753/e39f2032-fda3-11e6-9ba1-f6ba0a1b2b9f.jpg)

**Comment by juliocesar36** [comment]
nativeBase not work? that happens to me too...

**Comment by shivrajkumar** [comment]
![screen shot 2017-03-16 at 6 43 32 pm](https://cloud.githubusercontent.com/assets/17001305/23997716/84720610-0a78-11e7-9cea-0cbf12b370a7.png)
Seem to work fine

**Comment by avdept** [comment]
Getting lots of same errors. Any update on this?

**Comment by NickAlvesX** [comment]
`import { Container, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body } from 'native-base';
// Delete this row.
export default class Match extends Component {`

`import { Container, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body } from 'native-base';
export default class Match extends Component {`

**Comment by NickAlvesX** [comment]
```
import { Container, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body } from 'native-base'; 
// Just delete this row. 
export default class Match extends Component {
```
```

import { Container, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body } from 'native-base'; 
export default class Match extends Component {
```

**Comment by levsthings** [comment]
Today I tried using the [Segment](https://docs.nativebase.io/COMPONENTS.html#Segment) component and having the same problem.
Getting the error:
```Unexpected character " (4:0) ```

I tried other example components provided in the documentation afterwards and they are all giving the same error.  I've tried cleaning the examples up, deleting rows etc. as suggested but nothing seems to solve this issue. 

![nativebase-error](https://cloud.githubusercontent.com/assets/20170364/24713548/b36c3006-1a2e-11e7-8941-fc7a691496e9.png)

Additional Info:

**Versions**
```
"react": "~15.4.1",
"react-native": "0.42.3",
"native-base": "^2.1.1",
```
